.github/workflows/build-test-package.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       matrix:
         python-version: [37, 38, 39, 310]
         include:
-          - itk-python-git-tag: "v5.3rc03"
+          - itk-python-git-tag: "v5.3rc04"
```

Comment:
This tag does not exist yet. This probably results in it being ignored.

Reply:
> This tag does not exist yet. This probably results in it being ignored.

You are right that the C++ repo is not tagged, but `itk-python-git-tag` is used to reference the ITKPythonPackage wheel archive to download, which has been tagged for ITK v5.3rc04. We should be able to use this wheel for building external module wheels as well. In the meantime for C++ builds I have supplied the ITK commit tag used in the [ITKPythonPackage v5.3rc04 builds](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/de54e148584827cc3cb89898358510877762f1b7/CMakeLists.txt#L97).

[export itk-python-git-tag for ITKPythonPackage scripts](https://github.com/KitwareMedical/ITKUltrasound/blob/master/.github/workflows/build-test-package.yml#L171)

[use ITK_PACKAGE_VERSION to download corresponding wheel archive](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh#L50)

[ITKPythonPackage v5.3rc04 wheel artifacts](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/releases/tag/v5.3rc04)

With ITKPythonPackage wheels available we _should_ be able to build corresponding wheels for external modules.